.env.dev

```diff
@@ -1 +1,3 @@
-SERVER_PORT=5000
\ No newline at end of file
+SERVER_PORT=5000
+VITE_BACKEND_URL=http://localhost:8080
+VITE_FRONTEND_URL=http://localhost:3000
```

Comment:
we don't need front end url, we are out of control over it
front end runs wherever someone or something kicks it off.
and, besides, it's not used anywhere.

better suggestion is doing below:
- remove VITE_BACKEND_URL and VITE_FRONTEND_URL
- add SERVER_ADDRESS=localhost
- do to vite.config.js and add 
  ```js
  const serverAddress = process.env.SERVER_ADDRESS || 'localhost'
  ```
  then in line 15 replace localhost with `${serverAddress}`
- rename `.env.dev ` into `env.local` (that was mistake to call it dev in fact)
- update dev script in package.json by replacing `--mode dev` with `--mode local`
- from then on, stop using this
  ```js
  const API_BASE_URL = import.meta.env.VITE_BACKEND_URL;
  ```
  instead do 
  ```js
  fetch('/api/v1/.......')
  ```

Reply:
1)env.local was causing an error, so env.local-mode
2)also changed SERVER_PORT = 8080
3) in vite.config.js added  - api/v1....
proxy: {
        '/api': {
          target: `http://${serverAddress}:${serverPort}`,
          changeOrigin: true,
          rewrite: (path) => path.replace(/^\/api\/v1/, ''),
        },
      },

Reply:
you must have been doing something wrong.
it actually works fine with 
```json
"scripts": {
    "dev": "vite --open --port 3000 --mode local",
}
```
and with `.env.local` file.